modules/sbo-deploying-the-spring-petclinic-sample-application-power-z.adoc

```diff
@@ -0,0 +1,115 @@
+[id="sbo-deploying-the-spring-petclinic-sample-application-ibm-power-z_{context}"]
+= Deploying the Spring PetClinic sample application
+
+To deploy the Spring PetClinic sample application on an {product-title} cluster, you must use a deployment configuration and configure your local environment to be able to test the application.
+
+[discrete]
+.Procedure
+
+. Deploy the `spring-petclinic-rest` application with the `PostgresCluster` custom resource (CR) by running the following command in shell:
++
+[source,terminal]
+----
+$ oc apply -f - << EOD
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: spring-petclinic-rest
+  namespace: my-postgresql
+  labels:
+    app: spring-petclinic-rest
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: spring-petclinic-rest
+  template:
+    metadata:
+      labels:
+        app: spring-petclinic-rest
+    spec:
+      containers:
+        - name: application
+          image: quay.io/service-binding/spring-petclinic-rest:latest
+          env:
+          - name: SPRING_PROFILES_ACTIVE
+            value: postgresql,spring-data-jpa
+          - name: org.springframework.cloud.bindings.boot.enable
+            value: "true"
+          ports:
+          - name: http
+            containerPort: 9966
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: spring-petclinic-rest
+  namespace: my-postgresql
+spec:
+  ports:
+  - port: 80
+    targetPort: 9966
+  selector:
+    app: spring-petclinic-rest
+---
+EOD
+----
++
+The output verifies that the Spring PetClinic sample application is created and deployed:
++
+.Example output
+[source,terminal]
+----
+deployment.apps/spring-petclinic-rest created
+service/spring-petclinic-rest created
+----
++
+The output verifies that the database is created:
++
+[source,terminal]
+----
+$ oc get pods -n my-postgresql
```

Comment:
@ktania46  - this might have been missed - I believe the *oc get pods* command should be in a step similar to what you have in  modules/sbo-creating-a-postgresql-database-instance-power-z.adoc with the same command.